### PR TITLE
[hi] Localize Database as a service

### PR DESCRIPTION
**Pull request by sumit-158**
This PR localizes file content/hi/database_as_a_service.md in Hindi

solve #894

### CODE REVIEW

File: content/hi/database_as_a_service.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: डेटाबेस-एज़-ए-सर्विस (DBaaS)
+status: Completed
+category: प्रौद्योगिकी
+---
+
+## यह क्या है 
+
+डेटाबेस-ए-ए-सर्विस (DBaaS) एक [क्लाउड](/cloud_computing/) ऑपरेटर (सार्वजनिक या निजी) द्वारा प्रबंधित एक सेवा है जो पारंपरिक डेटाबेस प्रशासन कार्यों को करने के लिए एप्लिकेशन टीम की आवश्यकता के बिना अनुप्रयोगों का समर्थन करती है। DBaaS ऐप डेवलपर्स को डेटाबेस को अपडेट रखने के लिए विशेषज्ञ न होने या डेटाबेस एडमिनिस्ट्रेटर (DBA) को हायर किए बिना डेटाबेस का लाभ उठाने की अनुमति देता है।
```
Comment (jayesh-srivastava) [inline content/hi/database_as_a_service.md:9]: It should be 'डेटाबेस-एज़-ए-सर्विस'

**Review by jayesh-srivastava — CHANGES_REQUESTED** [review]

**Comment by sumit-158** [comment]
@jayesh-srivastava its ready to review

File: content/hi/database_as_a_service.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: डेटाबेस-एज़-ए-सर्विस (DBaaS)
+status: Completed
+category: प्रौद्योगिकी
+---
+
+## यह क्या है 
+
+डेटाबेस-एज़-ए-सर्विस (DBaaS) एक [क्लाउड](/cloud_computing/) ऑपरेटर (सार्वजनिक या निजी) द्वारा प्रबंधित एक सेवा है जो पारंपरिक डेटाबेस प्रशासन कार्यों को करने के लिए एप्लिकेशन टीम की आवश्यकता के बिना अनुप्रयोगों का समर्थन करती है। DBaaS ऐप डेवलपर्स को डेटाबेस को अपडेट रखने के लिए विशेषज्ञ न होने या डेटाबेस एडमिनिस्ट्रेटर (DBA) को हायर किए बिना डेटाबेस का लाभ उठाने की अनुमति देता है।
```
Comment (anubha-v-ardhan) [inline content/hi/database_as_a_service.md:9]:
```suggestion
डेटाबेस-एज़-ए-सर्विस (DBaaS) एक [क्लाउड](/cloud_computing/) ऑपरेटर (सार्वजनिक या निजी) द्वारा प्रबंधित एक सेवा है जो पारंपरिक डेटाबेस प्रशासन कार्यों को करने के लिए एप्लिकेशन टीम की आवश्यकता के बिना एप्लीकेशनों का समर्थन करती है। DBaaS ऐप डेवलपर्स को डेटाबेस को अपडेट रखने के लिए विशेषज्ञ न होने या डेटाबेस एडमिनिस्ट्रेटर (DBA) को नियुक्त किए बिना डेटाबेस का लाभ उठाने की अनुमति देता है।
```

File: content/hi/database_as_a_service.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या
+
+परंपरागत रूप से, ऑन-प्रिमाइसेस सेटअप में, संगठनों को नियमित रूप से डेटाबेस विस्तार को समायोजित करने के लिए अतिरिक्त भंडारण और प्रसंस्करण क्षमता में निवेश करना पड़ता है जो महंगा हो सकता है। इसके अतिरिक्त, डेवलपर्स आईटी इन्फ्रास्ट्रक्चर टीमों की मदद से डेटाबेस का प्रावधान और कॉन्फ़िगरेशन करते हैं, जिससे डेटाबेस-संचालित अनुप्रयोगों की तैनाती की गति धीमी हो जाती है। उन्हें लोड करने और निष्पादित करने में भी अधिक समय लगता है।
+
+## समाधान
+
+डीबीएएएस डेवलपर्स को सभी प्रशासन/प्रशासनिक कार्यों को क्लाउड-आधारित सेवा प्रदाता को आउटसोर्स करने की अनुमति देता है। सेवा प्रदाता यह सुनिश्चित करता है कि डेटाबेस सुचारू रूप से चल रहा है, जिसमें कॉन्फ़िगरेशन प्रबंधन, बैकअप, पैच, अपग्रेड, सेवा निगरानी, ​​​​और बहुत कुछ शामिल है, यह सब प्रबंधित करने के लिए उपयोगकर्ता के अनुकूल इंटरफेस के साथ। DBaaS डेटाबेस लागत को कम करते हुए संगठनों को एंटरप्राइज़-ग्रेड एप्लिकेशन को तेज़ी से विकसित करने में मदद करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/database_as_a_service.md:17]:
```suggestion
DBaaS डेवलपर्स के सभी प्रशासन/प्रशासनिक कार्यों को क्लाउड-आधारित सेवा प्रदाता को आउटसोर्स करने की अनुमति देता है। सेवा प्रदाता यह सुनिश्चित करता है कि डेटाबेस सुचारू रूप से चल रहा है, जिसमें कॉन्फ़िगरेशन प्रबंधन, बैकअप, पैच, अपग्रेड, सेवा निगरानी (service monitoring), ​​​​और बहुत कुछ प्रबंधित करने के लिए उपयोगकर्ता के अनुकूल इंटरफेस के साथ शामिल है। DBaaS डेटाबेस लागत को कम करते हुए संगठनों को एंटरप्राइज़-ग्रेड एप्लिकेशन को तेज़ी से विकसित करने में मदद करता है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
@sumit-158 Please also change the file name to `database-as-a-service.md`. We prefer keeping the file names same as that of English version.

**Comment by sumit-158** [comment]
@anubha-v-ardhan done!

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM!

Please wait for one more approval for this PR to be merged.
Thank you @sumit-158

**Review by Garima-Negi — APPROVED** [review]
LGTM!

**Comment by Garima-Negi** [comment]
will need @jayesh-srivastava 's approval to proceed with a merge

**Comment by Garima-Negi** [comment]
@jayesh-srivastava waiting on your approval on this one :)

**Review by jayesh-srivastava — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**